cmd/servicelog/post.go

```diff
@@ -87,14 +89,39 @@ func (o *PostCmdOptions) Validate() error {
 	return nil
 }
 
+func (o *PostCmdOptions) CheckServiceLogsLastHour() bool {
+	getAllMessages := false      // we need just manual entries
+	getInternalLogsOnly := false // we need all messages
+	numberOfHours := 1           // number of hours we need to wait for svc logs
+	response, err := FetchServiceLogs(o.ClusterId, getAllMessages, getInternalLogsOnly)
+	if err != nil {
+		log.Fatalf("failed to fetch service logs: %q", err)
+	}
+	timeStampToCompare := time.Now().Add(-time.Hour * time.Duration(numberOfHours))
+	for _, svclog := range response.Items().Slice() {
+		timestamp := svclog.Timestamp()
+		if timestamp.After(timeStampToCompare.UTC()) {
+			return true // prompt message for re-confirm
+		}
+	}
+	return false
+}
+
 func (o *PostCmdOptions) Run() error {
 	if err := o.Init(); err != nil {
 		return err
 	}
 	if err := o.Validate(); err != nil {
 		return err
 	}
-
+	if term.IsTerminal(int(os.Stdout.Fd())) && o.CheckServiceLogsLastHour() {
+		var resp string
+		fmt.Println("There is a service log within last 60 mins, are you sure you want to submit! Type YES to confirm")
+		fmt.Scan(&resp)
+		if resp != "YES" {
```

Comment:
I believe we already have a helper function that could be re-used here: https://github.com/openshift/osdctl/blob/cdfaf84d3593afed1797b42758f657baebdf04aa/pkg/utils/utils.go#L249-L264